packages/dcos-integration-test/extra/test_mesos.py

```diff
@@ -157,7 +158,11 @@ def test_if_ucr_app_runs_in_new_pid_namespace(dcos_api_session):
         marathon_framework_id = dcos_api_session.marathon.get('/v2/info').json()['frameworkId']
         app_task = dcos_api_session.marathon.get('/v2/apps/{}/tasks'.format(app['id'])).json()['tasks'][0]
 
-        content = dcos_api_session.mesos_sandbox_file(
-            app_task['slaveId'], marathon_framework_id, app_task['id'], ps_output_file)
+        # There is a race between the `app_task` starting and it writing its output to the `ps_output_file`.
```

Comment:
This comment doesn't really seem accurate. There's no race condition. It's just that:
1. task health just means the process it running. It's more of a "health check" than a "readiness check". If you were testing a readiness check you could make sure the file was written before the app responded as ready.
2. the task has to write to disk _after_ it starts running. So expecting it to be immediately available is a poor assumption.
3. in dcos-vagrant the host resources are often more oversubscribed and slower than on an AWS cluster.